### PR TITLE
Implement skeleton of simple profile pages

### LINKED ISSUE #1081: Implement my profile settings

### PR DESCRIPTION
**Pull request by eatonphil**
screen shot 2017-02-02 at 11 20 08 am
screen shot 2017-02-02 at 11 20 14 am
screen shot 2017-02-02 at 11 20 19 am
screen shot 2017-02-02 at 11 20 24 am
screen shot 2017-02-02 at 11 20 33 am
screen shot 2017-02-02 at 11 20 40 am

Closes #1081

### CODE REVIEW

**Comment by mparke** [comment]
Question re: display, why not have these fields as 1 form? Are these separate api endpoints?

**Comment by mparke** [comment]
Question re: password & auth, why not separate these forms into 2 tabs? have 1 context per tab

**Comment by eatonphil** [comment]
Short story to both: it's implemented that way currently.

File: karma.conf.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ module.exports = function(config) {
       module: {
         loaders: [
           {
+            test: /\.json$/,
+            loaders: ['json-loader'],
```
Comment (mparke) [inline karma.conf.js]: what uses the json-loader? moment-timezone?

Comment (eatonphil) [inline karma.conf.js]: yep

**Comment by eatonphil** [comment]
thanks!

**Pull request merged**